beefy-cli/Cargo.toml

```diff
@@ -18,6 +18,9 @@ structopt = "0.3.21"
 beefy-primitives = { git = "https://github.com/paritytech/substrate", branch = "master" }
 beefy-merkle-tree = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
+# Polkadot dependencies
+rococo-runtime = { git = "https://github.com/paritytech/polkadot", branch = "master" }
```

Comment:
Let's not depend on the entire polkadot repo just for this. Can you please just hardcode that? The leaf data structure is hardcoded as well anyway (coming from substrate primitives), it's not like we will be automatically able to support newer versions.

Reply:
done, ptal